tools/checkpoint_conversion/convert_mistral_checkpoints.py

```diff
@@ -300,7 +300,7 @@ def main(_):
     print("-> Saved the model weights in float16")
 
     # === Save the model config ===
-    keras_nlp_config["dtype"] = "bfloat16"
+    keras_nlp_config.pop("dtype")  # We don't want a default dtype
```

Comment:
I would actually rename `keras_nlp_config` -> `backbone_kwargs`, and save using
```
keras_nlp.src.utils.preset_utils.save_to_preset(
    keras_nlp_model, preset
)
keras_nlp.src.utils.preset_utils.save_to_preset(
    keras_nlp_tokenizer, preset, config_filename="tokenizer.json"
) 
```
Those will do the right thing and always call `model.get_config()` to create the `config.json` file (which does not include `dtype` for this reason). No need to regenerate presets if things look good in testing.

Reply:
Done, uploaded the new presets to Kaggle.